portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -68,4 +76,14 @@ public String getParameter(HttpServletRequest request, String parameter, String
     String value = request.getParameter(parameter);
     return value == null ? defaultValue : value;
   }
+
+  static class Comment {
```

Comment:
same comment as the `User` in the other file - either bring this class outside, or make it private instead of static.